### PR TITLE
Systems_baltimoreadded

### PR DESCRIPTION
**Pull request by jedweeks**
still lacking bolt feed for Baltimore, Lime, Jump, Spin added.

### CODE REVIEW

File: systems.csv [hunk 1]
```diff
@@ -202,6 +202,7 @@ US,Indy - Pacers Bikeshare,"Indianapolis, IN",bcycle_pacersbikeshare,https://www
 US,Jackson County,"Jackson, MI",bcycle_jacksoncounty,https://jacksoncounty.bcycle.com,https://gbfs.bcycle.com/bcycle_jacksoncounty/gbfs.json
 US,JerseyBike,"US",nextbike_nj,https://www.hudsonbikeshare.com/xx/hoboken/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_nj/gbfs.json
 US,JUMP Austin,"Austin, TX",jump_austin,https://jump.com/,https://atx.jumpbikes.com/opendata/gbfs.json
+US,JUMP Baltimore,"Baltimore, MD",jump_baltimore,https://jump.com/,https://gbfs.uber.com/v1/balts/free_bike_status.json
```
Comment (jcn) [inline systems.csv]: This should point to the GBFS auto-discovery URL not the free_bike_status feed URL

File: systems.csv [hunk 2]
```diff
@@ -246,6 +248,7 @@ US,Skybike West Palm Beach,"West Palm Beach Florida, US",nextbike_wb,https://sky
 US,Sobi Long Beach,"Long Beach, NY",sobi_long_beach,http://sobilongbeach.com/,http://sobilongbeach.com/opendata/gbfs.json
 US,Spartanburg BCycle,"Spartanburg, SC",bcycle_spartanburg,https://spartanburg.bcycle.com,https://gbfs.bcycle.com/bcycle_spartanburg/gbfs.json
 US,Spin Chicago,"Chicago, IL",spin chicago,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/chicago/gbfs.json
+US,Spin Baltimore, "Baltimore, MD,spin baltimore,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/baltimore/free_bike_status.json
```
Comment (jcn) [inline systems.csv]: This one should also be pointing at the gbfs feed not the free_bike_status feed.

**Review by jcn — DISMISSED** [review]

File: systems.csv (hunk 2 above)
Comment (jedweeks) [inline systems.csv]:
```suggestion
US,Spin Baltimore, "Baltimore, MD,spin baltimore,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/baltimore/gbfs.json

File: systems.csv (hunk 1 above)
Comment (jedweeks) [reply systems.csv]:
```suggestion
US,JUMP Baltimore,"Baltimore, MD",jump_baltimore,https://jump.com/,https://gbfs.uber.com/v1/balts/gbfs.json
```

**Comment by heidiguenin** [comment]
Hey @jedweeks could you revise your review request to me? I'll go ahead and take care of it, but it seems I can't since only @jcn is tagged.

**Review by heidiguenin — APPROVED** [review]

File: systems.csv (hunk 1 above)
Comment (heidiguenin) [inline systems.csv]: This is still pointing to free_bike_status.json instead of gbfs.json

Comment (heidiguenin) [reply systems.csv]: @jedweeks the suggested change hasn't been committed yet - can you do that?

**Pull request merged**